### ISSUE TITLE
Adding Laos and Khmer. - Source is Youtube.

### ISSUE THREAD

**Issue by Auratus** [larsyencken/wide-language-index#33]
Laos - https://youtu.be/SVD8v2vaJ8k?t=22s

Khmer - https://www.youtube.com/watch?v=6WWtEQYj28Y

You could use Youtube to MP3. If possible.

**Comment by bcaller** [comment]
Yeah we should add a youtube-dl downloader for downloading videos as it is easy to find content there.
I have already worked on something similar: https://gist.github.com/bcaller/58e4a14d334ac998267f7c8cbfb876d0
It saves audio for all the videos in the wikitongues playlist that contain just a single language. The problem is that a lot of the recordings in the wikitongues project are not very fluent.
